lib/acquia-http-hmac/rack_authenticate.rb

```diff
@@ -23,6 +23,8 @@ def call(env)
         return unauthorized if auth_header.empty?
 
         attributes = Acquia::HTTPHmac::Auth.parse_auth_header(auth_header)
+        # Put the id into a variable regardless of authentication so we can read/log it later
+        env['REQUESTED_HMAC_ID'] = attributes[:id]
```

Comment:
The other environment variables specific to this implementation of the spec are prefixed with ACQUIA-HTTP-HMAC, any reason not to do that here?